src/main.jsx

```diff
@@ -6,6 +6,9 @@ import 'firebase/auth';
 
 import App from './component/App';
 import config from './config/config';
+import test from './routing/routingtest';
+
+test();
```

Comment:
make note to comment this out/remove in production

Reply:
noted it on trello